public/index.html

```diff
@@ -38,5 +39,13 @@
       To create a production bundle, use `npm run build` or `yarn build`.
     -->
     <script src="https://apis.google.com/js/platform.js"></script>
+    <script>
+      <!-- track when users exit the site after starting a query -->
+      window.onbeforeunload = function () {
+        if(window.localStorage.getItem('KF_GA_TIMING_INIT_FILE_QUERY_TO_CAVATICA_COPY') || window.localStorage.getItem('KF_GA_TIMING_INIT_FILE_QUERY_TO_DOWNLOAD') ){
+          window.ga('send', 'event', 'File Repo', 'Query Abandoned', 'exited site', 3);
+        }
+      }
+    </script>
```

Comment:
does this have to be in the html? why not have this in `src/services/analyticsTracking.js`?

Reply:
couldn't get it to fire from the analytics tracking service. I think it's because the event is fired after the page unloads all it's external resources before exiting